src/view/components/common/main-navigation/main-navigation.html

```diff
@@ -4,13 +4,13 @@
             <accordion-group [heading]="menu.tabName" [icon]="menuIcons[menu.tabName]">
                 <div class="main-nav__submenu">
                     <template ngFor let-menuItem [ngForOf]="menu.menuItems">
-                        <a [routerLink]="[menuItem.id]" *ngIf="menuItem.url && !menuItem.angular && !menuItem.ajax">
+                        <a [routerLink]="['/c/' + menuItem.id]" *ngIf="menuItem.url && !menuItem.angular && !menuItem.ajax">
```

Comment:
Maybe is better concat this into typescript code instead.

Maybe into RoutingService, something like (when we iterate the menu items)
...
menuItem.routerLink = '/c/' + menuItem.id;
...

and then use it into the html:

<a [routerLink]="[menuItem.routerLink]" *ngIf="menuItem.url && !menuItem.angular && !menuItem.ajax">